### DISCUSSION TITLE
Advance Test case 2

### DISCUSSION THREAD

**Discussion by faccoco** [SUSTech-CS305-Fall22/CS305-Project-Skeleton#81] [Q&A]
进行advance test的第二个测试时，我注意到接收方需要接收两个chunk。我们小组的代码逻辑是完成一个chunk的接受立刻写入并生成一个文件，但test代码似乎会在生成一个文件之后立即停止运行，请问这种情况该怎么办呢

**Comment by IcyConst** [comment]
在程序中改成接收完所有chunk再生成。这个之前discussion里有提过，估计必须是得这样改了。
